lib/tasks/publishing_api.rake

```diff
@@ -5,15 +5,30 @@ require "gds_api/publishing_api"
 require "gds_api/publishing_api/special_route_publisher"
 
 namespace :publishing_api do
-  def publishing_api
-    GdsApi::PublishingApiV2.new(
-      Plek.new.find("publishing-api"),
-      bearer_token: ENV["PUBLISHING_API_BEARER_TOKEN"] || "example",
+  desc "Publish special routes via publishing api"
+  task publish_special_routes: :environment do
+    logger = Logger.new(STDOUT)
+    special_route_publisher = GdsApi::PublishingApi::SpecialRoutePublisher.new(
+      logger: logger,
+      publishing_api: GdsApi.publishing_api,
```

Comment:
Shouldn't this be `publishing_api: GdsApi.publishing_api_v2`?

Reply:
Not any more: https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/publishing_api_v2.rb#L11

Reply:
Ah, I see: https://github.com/alphagov/gds-api-adapters/pull/969

Ok, great!